cardano-constitution/test/Cardano/Constitution/Validator/Data/GoldenTests/sorted.cbor.size.golden

```diff
@@ -1 +1 @@
-2167
\ No newline at end of file
+2164
```

Comment:
How come the size was reduced? I would expect the benchmarks in this PR to have their size increased by a maximum of 5.